flink-python/pyflink/table/table_environment.py

```diff
@@ -76,14 +76,24 @@ class TableEnvironment(object):
 
     __metaclass__ = ABCMeta
 
-    def __init__(self, j_tenv, is_blink_planner, serializer=PickleSerializer()):
+    def __init__(self, j_tenv, serializer=PickleSerializer()):
```

Comment:
Are the changes in this class relevant to the `MLEnvironment` and `MLEnvironmentFactory`?

Reply:
Yes, it will be more friendly to create the corresponding table environment in `MLEnvironmentFactory.get_default()`, e.g., we don not need to pass the is_blink_planner variable when creating `StreamTableEnvironment`.

Reply:
Do we expect others to construct `TableEnvironment`? Isn't this a public constructor?

Reply:
It is an internal constructor which takes a java table environment object as a parameter. For users, the `StreamTableEnvironment.create()` is called to create the Environment.

Reply:
Got it. Thanks for the explanation.